box-share-sdk/src/main/java/com/box/androidsdk/share/utils/ShareSDKTransformer.java

```diff
@@ -23,7 +26,7 @@
 /**
  * A utility class for transforming BoxResponses into PresenterData for InviteCollaboratorsShareVM.
  */
-public class InviteCollabsTransformer {
+public class ShareSDKTransformer {
```

Comment:
The name is too generic. Also, is the comment accurate?

Reply:
The comment is not accurate. This class is responsible for doing the transformation for all the VMs. I could create as separate transformer for each VM.